sc_cost_meter/app.py

```diff
@@ -1,42 +1,27 @@
-import json
+import logging
+import sc_cost_meter.utils as utils
 
-# import requests
+from datetime import datetime, timedelta
 
+log = logging.getLogger(__name__)
+log.setLevel(logging.DEBUG)
 
 def lambda_handler(event, context):
-    """Sample pure Lambda function
-
-    Parameters
-    ----------
-    event: dict, required
-        API Gateway Lambda Proxy Input Format
-
-        Event doc: https://docs.aws.amazon.com/apigateway/latest/developerguide/set-up-lambda-proxy-integrations.html#api-gateway-simple-proxy-for-lambda-input-format
-
-    context: object, required
-        Lambda Context runtime methods and attributes
-
-        Context doc: https://docs.aws.amazon.com/lambda/latest/dg/python-context-object.html
-
-    Returns
-    ------
-    API Gateway Lambda Proxy Output Format: dict
-
-        Return doc: https://docs.aws.amazon.com/apigateway/latest/developerguide/set-up-lambda-proxy-integrations.html
-    """
-
-    # try:
-    #     ip = requests.get("http://checkip.amazonaws.com/")
-    # except requests.RequestException as e:
-    #     # Send some context about this error to Lambda Logs
-    #     print(e)
-
-    #     raise e
-
-    return {
-        "statusCode": 200,
-        "body": json.dumps({
-            "message": "hello world",
-            # "location": ip.text.replace("\n", "")
-        }),
+  synapse_ids = utils.get_marketplace_synapse_ids()
```

Comment:
Just a thought:  If you gathered the synapse IDs, customer IDs, and product codes in this initial query, then you wouldn't have to do the additional queries (get_marketplace_customer_id() and get_marketplace_product_code()) below.

Reply:
good idea.  will implement in follow up PR

Reply:
done in https://github.com/Sage-Bionetworks-IT/lambda-sc-cost-meter/pull/7